File: Samples/Bit6ChatDemo/ChatDemo/ImageAttachedViewController.m
//
//  ImageAttachedViewController.m
//  Bit6ChatDemo
//
//  Created by Carlos Thurber Boaventura on 06/05/14.
//  Copyright (c) 2014 Bit6. All rights reserved.
//

#import "ImageAttachedViewController.h"

@interface ImageAttachedViewController ()

@property (weak, nonatomic) IBOutlet Bit6ImageView *imageView;

@end

@implementation ImageAttachedViewController

- (void) viewWillAppear:(BOOL)animated
{
    [self.navigationController setToolbarHidden:YES animated:YES];
}

- (void) viewDidLoad
{
    self.navigationItem.prompt = [NSString stringWithFormat:@"Logged as %@",Bit6.session.userIdentity.displayName];
    self.hidesBottomBarWhenPushed = YES;
    
    self.imageView.message = self.message;
    [super viewDidLoad];
}

@end

File: Samples/Bit6ChatDemo-Swift/Bit6ChatDemo-Swift/ImageAttachedViewController.swift
//
//  ImageAttachedViewController.swift
//  Bit6ChatDemo-Swift
//
//  Created by Carlos Thurber on 11/23/14.
//  Copyright (c) 2014 Bit6. All rights reserved.
//

import UIKit
import Bit6

class ImageAttachedViewController: UIViewController {

    var message : Bit6Message!

    @IBOutlet var imageView: Bit6ImageView!
    
    override func viewDidLoad() {
        if let userIdentity = Bit6.session().activeIdentity {
            self.navigationItem.prompt = "Logged as \(userIdentity.displayName)"
        }
        self.imageView.message = self.message
        self.hidesBottomBarWhenPushed = true
    }
    
    override func viewWillAppear(animated: Bool) {
        self.navigationController?.setToolbarHidden(true, animated: true)
    }
    
}
